browser/components/CodeEditor.js

```diff
@@ -163,15 +164,16 @@ export default class CodeEditor extends React.Component {
     this.editor.setCursor(cursor)
   }
 
-  handleDrop (e) {
+  handleDropImage (e) {
     e.preventDefault()
-    let path = e.dataTransfer.files[0].path
-    let filename = path.match(".+/(.+?)\.[a-z]+([\?#;].*)?$")[1]
-    let imageMd = "![" + filename + "](" + path + ")"
-    this.insertImage(this.editor.getInputField(), imageMd)
+    let imagePath = e.dataTransfer.files[0].path
+    let filename = path.basename(imagePath)
```

Comment:
Hmm, I think it works on macOS. But on windows, path delimiters may be backslash.
I think this way is better.